packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { css, cx } from 'emotion';
+import { stylesFactory, useTheme } from '../../themes';
+
+export interface InfoBoxProps {
```

Comment:
just name this type Props   (very rare cases where we have multiple components in a file do we prefix the props type)

Reply:
Yes, agreed